tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,209 @@
+
+nsb --> auditQ(Audit Queue)
+nsb --> errorQ(Error Queue)
+
+subgraph ServiceControl
+  scinstance
+  scmonitoring
+end
+
+auditQ --> scinstance(ServiceControl Instance)
+errorQ --> scinstance
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+scinstance --> pulse
+
+```
+
+![the above renders like this](component-overview.png)
+
+Every system being monitored will push data to three main queues:
+
+- Monitoring: Each endpoint will gather and send metric data to this queue. This data holds information about the health and performance of an endpoint.
+- Audit: When an endpoint successfully processes a message, a copy of the message will be forwarded to this queue. The audited message will have additional headers which contain data about the processing of the message.
+- Error: When an endpoint fails to process a message (after exhausting automated retry attenpts), a copy of the message is fowarded to this queue. The failed message will have additional headers which contain data about the failed attempt to process the message.
+
+The core NServiceBus package can be configured to send messages to the Audit and Error queues. In order to send data to the Monitoring queue, two additional packages are required:
+
+- NServiceBus.Metrics: hooks into the NServiceBus pipeline to gather and expose metrics.
+- NServiceBus.Metrics.ServiceControl: buffers the metric data generated by NServiceBus.Metrics and periodically sends it to the monitoring queue. This package also instruments outgoing messages with additional headers that are used to estimate queue length.
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system. A monitoring system back-end consists of two different components:
+
+- ServiceControl instance: collects messages from the audit and error queues. Error messages are grouped together and can be retried in bulk.
+- Monitoring instance: collects data from the monitoring queue about all of the endpoints in an NServiceBus system. 
+
+Both of these instance types can be set-up and managed with the ServiceControl Management Utility.
+
+ServicePulse is a web application that connects to ServiceControl and provides production monitoring and recoverability. It contains pages to visualize the monitoring data in a Monitoring instance. It can also be used to review and manage failed messages that are managed by a ServiceControl instance.
+
+
+
+## Setting up Particular Service Platform
+
+In order to configure an environment for monitoring with the Particular Service Platform you will need to install and configure the components in the order listed.
+
+
+### Install ServiceControl
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system. The main installer includes a desktop utility called the ServiceControl Management Utility which can be used to create and manage ServiceControl and Monitoring instances.
```

Comment:
This is duplication with line [56](https://github.com/Particular/docs.particular.net/pull/3401/files#diff-f4990acf25b99633a56721d261c3f459R56)